rust/rope/src/tree.rs

```diff
@@ -46,18 +46,22 @@ pub trait NodeInfo: Clone {
     /// The identity of the monoid. Need not be implemented because it
     /// can be computed from the leaf default.
     ///
-    /// This is hear to demonstrate that this is a monoid.
+    /// This is here to demonstrate that this is a monoid.
     fn identity() -> Self {
         Self::compute_info(&Self::L::default())
     }
 
-    /// The interval covered by this node. The default impl is sufficient for most types,
-    /// but interval trees may need to override it.
+    /// The interval covered by the first `len` base units of this node. The
+    /// default impl is sufficient for most types, but interval trees may need
+    /// to override it.
     fn interval(&self, len: usize) -> Interval {
         Interval::new(0, len)
     }
 }
 
+/// A trait for the leaves of trees of type [Node].
```

Comment:
So I've played around with this, and to get this link to work I've changed this to ```[`Node`](struct.Node.html)```.

I'm not sure if you've tried this, but you can build and open the docs with `cargo doc --open`, which will show you how everything renders.

Reply:
I works for me, but I'm using rust nightly... I will add explicit links then.

Reply:
Ah yea okay, I remember seeing an RFC to support your style, didn't know what stage it was at.